ScoreSaber/Core/ReplaySystem/ReplayLoader.cs

```diff
@@ -30,12 +30,13 @@ public ReplayLoader(PlayerDataModel playerDataModel, MenuTransitionsHelper menuT
             _environmentsListModel = environmentsListModel;
         }
 
-        public async Task Load(byte[] replay, BeatmapLevel beatmapLevel, BeatmapKey beatmapKey, GameplayModifiers modifiers, string playerName) {
+        public async Task Load(byte[] replay, BeatmapLevel beatmapLevel, BeatmapKey beatmapKey, GameplayModifiers modifiers, string playerName, bool isUsersReplay = false) {
```

Comment:
I feel that it would be worse to workaround getting if its the users replay once inside the replay, but theres no player data inside the replay so wouldnt this be the only option of knowing if its the users own replay?

Reply:
i could set it earlier if thats preferred